docs/contribute/guidelines-code/categories.md

```diff
@@ -17,7 +17,7 @@ For each area of the codebase, there are established and favored programming lan
 - **CLI** - Node.js, TypeScript
 - **Desktop UI** - Node.js, JavaScript
 - **APIs** - C, Assembler, Java, Spring
-- **API Mediation Layer** - Java, Spring
+- **API Mediation Layer** - Java, Spring, JavaScript
```

Comment:
Please use the Docusaurus admonition syntax:
https://docusaurus.io/docs/next/markdown-features/admonitions